modules/pytket-qiskit/pytket/extensions/qiskit/backends/ibmq_emulator.py

```diff
@@ -46,7 +47,6 @@
 
 if TYPE_CHECKING:
     from pytket.predicates import Predicate  # type: ignore
-    from pytket.passes import BasePass  # type: ignore
```

Comment:
this line  can remain here as far as i can tell, just need quotes around the annotation